packages/lambdas/src/ehr/healthie/link-patient.ts

```diff
@@ -22,6 +22,10 @@ const waitTimeInMillis = parseInt(waitTimeInMillisRaw);
 export const handler = Sentry.AWSLambda.wrapHandler(async (event: SQSEvent) => {
   capture.setExtra({ event, context: lambdaName });
 
+  if (isNaN(waitTimeInMillis)) {
+    throw new MetriportError(`Invalid WAIT_TIME_IN_MILLIS: ${waitTimeInMillisRaw}`);
+  }
```

Comment:
Nice! Prob for another PR, but we could prob create a `getEnvVarAsInt` as this is common and we only now started validating those numbers. Non-blocking!